iquip/apps/scheduler.py

```diff
@@ -0,0 +1,307 @@
+"""App module for showing the experiment list."""
+
+from typing import Optional, Tuple
+
+from PyQt5.QtGui import QPainter
+from PyQt5.QtCore import Qt, QObject, QAbstractListModel, QModelIndex, QMimeData
+from PyQt5.QtWidgets import (
+    QStyleOptionViewItem, QWidget, QLayout, QLabel, QListView,
+    QPushButton, QHBoxLayout, QVBoxLayout, QAbstractItemDelegate, QAction
+)
+
+import qiwis
+from iquip.protocols import ExperimentInfo
+
+def delete_items_of_layout(layout: QLayout):
+    """Auxiliary function for deleting components in the layout.
+
+    Args:
+        layout: The layout whose elements are intended to be deleted.
+    """
+    if layout is not None:
```

Comment:
If you expect the case where `layout` is `None`, use type hint `layout: Optional[QLayout]`. Otherwise, this `if` statement is unnecessary.